main.tf

```
@@ -129,12 +129,34 @@ data "aws_iam_policy_document" "malware_protection_policy" {
   }
 }
 
+data "aws_iam_policy_document" "bucket_key_encryption_policy_enforced" {
+
+  statement {
+    sid = "EnforceBucketKeyEncryption"
+    principals {
+      type        = "*"
+      identifiers = ["*"]
+    }
+    effect  = "Deny"
+    actions = ["s3:PutObject"]
+    resources = [
+      "${aws_s3_bucket.default.arn}/*"
+    ]
+    condition {
+      test     = "StringNotEquals"
+      variable = "s3:x-amz-server-side-encryption-aws-kms-key-id"
+      values   = [var.kms_key_arn]
+    }
+  }
+}
```

```suggestion
data "aws_iam_policy_document" "bucket_key_encryption_policy_enforced" {
  statement {
    sid       = "EnforceBucketKeyEncryption"
    actions   = ["s3:PutObject"]
    effect    = "Deny"
    resources = ["${aws_s3_bucket.default.arn}/*"]

    principals {
      type        = "*"
      identifiers = ["*"]
    }

    condition {
      test     = "StringNotEquals"
      variable = "s3:x-amz-server-side-encryption-aws-kms-key-id"
      values   = [var.kms_key_arn]
    }
  }
}
```

Cleaned up the data resource:

* Removed empty line at the top
* Grouped single lines and blocks together, putting single lines on top
* Made `resources` one line - since it only has a single element we do not decrease readability
* Kept `conditions` at the bottom